### PR TITLE
Fix EZP-24158: Need to clear cache manually after platform install

### PR DESCRIPTION
**Pull request by andrerom**
https://jira.ez.no/browse/EZP-24158

Reproduced with ezpublish-docker VM.

*Context: This was the parts I skipped when I first added cache clearing by copying what CacheClearing command is doing, part still skipped is cache warmup. Maybe we could rather just reuse the cache clearing command here, afaik there is a way to do that but I don't remember how and I think I remember it being rather hackish looking but might be wrong.*

### CODE REVIEW

File: eZ/Bundle/PlatformInstallerBundle/src/Command/InstallPlatformCommand.php [hunk 1]
```diff
@@ -153,7 +158,18 @@ private function cacheClear( OutputInterface $output )
         }
 
         $output->writeln( sprintf( 'Clearing cache for directory <info>%s</info>', $this->cacheDir ) );
+        $oldCacheDir = $this->cacheDir . '_old';
+
+
```
Comment (crevillo) [inline eZ/Bundle/PlatformInstallerBundle/src/Command/InstallPlatformCommand.php]: Two blank lines?

**Comment by crevillo** [comment]
+1 with the two blank lines remark

**Comment by andrerom** [comment]
Verified on ezpublish-docker and behat now, review ping @bdunogier @lolautruche 
Also ready for review: https://github.com/ezsystems/ezplatform/issues/14

**Comment by lolautruche** [comment]
+1

**Comment by andrerom** [comment]
Thanks for the reviews guys :)

**Pull request merged**